samples/net/sockets/echo_client/boards/atsamr21_xpro.conf

```
@@ -1,23 +1,31 @@
 #
-# Copyright (c) 2020-2021, Gerson Fernando Budke
 # Copyright (c) 2019, Benjamin Valentin
+# Copyright (c) 2020-2023, Gerson Fernando Budke
 #
 # SPDX-License-Identifier: Apache-2.0
 #
 
-# Reduced buffers to fit into SAMR21 SoC
+# Reduced configs to fit into SAMR21 SoC
 CONFIG_CPP=n
 
-CONFIG_NET_PKT_RX_COUNT=6
-CONFIG_NET_PKT_TX_COUNT=6
-CONFIG_NET_BUF_RX_COUNT=6
-CONFIG_NET_BUF_TX_COUNT=6
-CONFIG_NET_MAX_CONTEXTS=4
+CONFIG_NET_PKT_RX_COUNT=5
+CONFIG_NET_PKT_TX_COUNT=5
+CONFIG_NET_BUF_RX_COUNT=5
+CONFIG_NET_BUF_TX_COUNT=5
+CONFIG_NET_MAX_CONTEXTS=2
 CONFIG_NET_MAX_CONN=1
 CONFIG_NET_MAX_ROUTES=1
 CONFIG_NET_MAX_NEXTHOPS=1
 
-CONFIG_SHELL_STACK_SIZE=768
-CONFIG_SHELL_CMD_BUFF_SIZE=64
+CONFIG_SHELL_STACK_SIZE=384
+CONFIG_SHELL_CMD_BUFF_SIZE=16
```

With this limitation commands such as `ieee802154 get_pan_id` cannot be issued.

I tested with the following and was able to perform [interop testing with RIOT](https://github.com/RIOT-OS/Release-Specs/tree/master/08-interop#task-11---udp-exchange-between-iotlab-m3-and-zephyr).  Maybe `CONFIG_SHELL_STACK_SIZE` can be reduced but this compiles and passes.
```suggestion
CONFIG_SHELL_STACK_SIZE=512
CONFIG_SHELL_CMD_BUFF_SIZE=32
```

---

Rollback `SHELL_STACK_SIZE=768` and applied `SHELL_CMD_BUFF_SIZE=32` as suggested